controllers/activemqartemisaddress_controller_test.go

```diff
@@ -975,12 +975,13 @@ func QueueStatAssertionInPod(brokerCrName string, podName string, namespace stri
 		Fail("function should be called with an existing cluster")
 	}
 
+	podKey := types.NamespacedName{
+		Name:      podName,
+		Namespace: namespace,
+	}
+	pod := &corev1.Pod{}
 	Eventually(func(g Gomega) {
-		key := types.NamespacedName{Name: namer.CrToSS(brokerCrName), Namespace: namespace}
-		sfsFound := &appsv1.StatefulSet{}
-
-		g.Expect(k8sClient.Get(ctx, key, sfsFound)).Should(Succeed())
-		g.Expect(sfsFound.Status.ReadyReplicas).Should(BeEquivalentTo(1))
```

Comment:
Is the following check removed on purpose?
```
g.Expect(sfsFound.Status.ReadyReplicas).Should(BeEquivalentTo(1))
```

Reply:
yes, so that it can be used on a CR with size N != 1